Sources/HttpPipelineTestSupport/HttpPipelineTestSupport.swift

```diff
@@ -7,15 +7,15 @@ import XCTest
 extension Response: Snapshot {
   public typealias Format = String
 
-  public static var snapshotFileExtension: String? {
-    return "response.txt"
+  public static var snapshotPathExtension: String? {
+    return "Response.txt"
   }
 
   public var snapshotFormat: String {
     let top = """
 Status \(self.status.rawValue) (\(self.status))
 Headers: [
-  \(self.headers.map { $0.description }.joined(separator: "\n  "))
+  \(self.headers.sorted(by: \.description).map { $0.description }.joined(separator: "\n  "))
```

Comment:
sorting headers, and using a new extension on array that can sort by key paths into comparables. this should be bridged with lenses in the prelude, but didn't wanna mess with that now